docs/source/onnx_dynamo.rst

```diff
@@ -0,0 +1,151 @@
+  - **ONNX Export Options**: :class:`ExportOptions` has a set of options that control the export process.
+  - **ONNX Registry**: :class:`OnnxRegistry` is the registry of ONNX operators and functions.
+  - **FX Graph Extractor**: :class:`FXGraphExtractor` extracts the FX graph from the PyTorch model.
+  - **Fake Mode**: :class:`ONNXFakeContext` is a context manager that enables fake mode for large scale models.
+  - **ONNX Export Output**: :class:`ExportOutput` is the output of the exporter that contains the exported ONNX graph and diagnostics.
+  - **ONNX Export Output Serializer**: :class:`ExportOutputSerializer` serializes the exported model to a file.
+  - **ONNX Diagnostic Options**: :class:`DiagnosticOptions` has a set of options that control the diagnostics emitted by the exporter.
+
+Dependencies
+------------
+
+The ONNX exporter depends on extra Python packages not available through PyTorchs's `requirements.txt` file:
+
+  - `ONNX <https://onnx.ai>`_
+  - `ONNX Script <https://onnxscript.ai>`_
+
+They can be installed through `pip <https://pypi.org/project/pip/>`_ utility as follows:
+
+.. code-block:: bash
+
+  pip install onnx onnxscript-preview    # TODO: Replace with `onnxscript` when it is official
+
+A simple example
+----------------
+
+See below a demonstration of exporter API in action with a simple image classifier model as example:
+
+.. code-block:: python
+
+  import torch
+
+  class MNISTModel(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.conv1 = torch.nn.Conv2d(1, 32, 3, 1, bias=False)
+        self.conv2 = torch.nn.Conv2d(32, 64, 3, 1, bias=False)
+        self.fc1 = torch.nn.Linear(9216, 128, bias=False)
+        self.fc2 = torch.nn.Linear(128, 10, bias=False)
+
+    def forward(self, tensor_x: torch.Tensor):
+        tensor_x = self.conv1(tensor_x)
+        tensor_x = torch.nn.functional.sigmoid(tensor_x)
+        tensor_x = self.conv2(tensor_x)
+        tensor_x = torch.nn.functional.sigmoid(tensor_x)
+        tensor_x = torch.nn.functional.max_pool2d(tensor_x, 2)
+        tensor_x = torch.flatten(tensor_x, 1)
+        tensor_x = self.fc1(tensor_x)
+        tensor_x = torch.nn.functional.sigmoid(tensor_x)
+        tensor_x = self.fc2(tensor_x)
+        output = torch.nn.functional.log_softmax(tensor_x, dim=1)
+        return output
+
+  tensor_x = torch.rand((64, 1, 28, 28), dtype=torch.float32)
+  export_output = torch.onnx.dynamo_export(MNISTModel(), tensor_x)
+
+As the code above shows, all you need is to provide ``torch.onnx.dynamo_export`` with an instance of the model and its input.
+The exporter will then return an instance of ``torch.onnx.ExportOutput`` that contains the exported ONNX graph along with extra information.
+
+The in-memory model (type: onnx.ModelProto) is available at ``export_output.model_proto`` following `ONNX IR spec <https://github.com/onnx/onnx/blob/main/docs/IR.md>`_,
```

Comment:
The mentioning of `ONNX IR spec` is a little unexpected. Potentially

```suggestion
The in-memory model available at ``export_output.model_proto`` is an ``onnx.ModelProto`` object in compliance with the `ONNX IR spec <https://github.com/onnx/onnx/blob/main/docs/IR.md>`_.
```

or something better?

Reply:
LMK what you think of the new version